DBLP Computer Science Bibliography.js

```diff
@@ -2,77 +2,105 @@
 	"translatorID": "625c6435-e235-4402-a48f-3095a9c1a09c",
 	"label": "DBLP Computer Science Bibliography",
 	"creator": "Adam Crymble, Sebastian Karcher, Philipp Zumstein",
-	"target": "^https?://(www\\.)?(dblp(\\.org|\\.uni-trier\\.de/|\\.dagstuhl\\.de/)|informatik\\.uni-trier\\.de/\\~ley/)",
-	"minVersion": "1.0.0b4.r5",
+	"target": "^https?://(www\\.)?(dblp\\d?(\\.org|\\.uni-trier\\.de/|\\.dagstuhl\\.de/))",
+	"minVersion": "3.0",
 	"maxVersion": "",
 	"priority": 100,
 	"inRepository": true,
 	"translatorType": 4,
-	"browserSupport": "gcsv",
-	"lastUpdated": "2016-09-11 19:41:55"
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2019-10-16 21:02:38"
 }
 
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2008-2019 Adam Crymble, Sebastian Karcher, Philipp Zumstein
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
```

Comment:
@acrymble AGPL is the standard license for all Zotero translator. Can you say here, that this is fine for you?

Reply:
@zuphilip looking at [git blame](https://github.com/zotero/translators/blame/master/DBLP%20Computer%20Science%20Bibliography.js) between the two of us I think we have changed every line of this translator. I think it's fair to remove @acrymble as a contributor or at least move ahead with the license without.

Reply:
Okay, I changed the license and (blaming) creator line in the translator accordingly. The contributions of him will be recognized forever by git's history!